helpers/util.sh

```diff
@@ -133,7 +133,7 @@ function buildmw {
     GIT_URL="$1"
     shift
     GIT_BRANCH=""
-    if [ "$1" != "" ]; then
+    if [[ "$1" != *.spec ]]; then
```

Comment:
It doesn't work with "..." :) hence the double [[ ]] https://viewsby.wordpress.com/2013/09/06/bash-string-ends-with/